src/components/AddAsset/AddAssetTokenTab.vue

```diff
@@ -0,0 +1,120 @@
+<template>
+  <div class="add-asset-token">
+    <div v-if="!tokenDetailsPageOpened" class="add-asset-token__page">
+      <search-input
+        autofocus
+        v-model="search"
+        :placeholder="t(`addAsset.searchInputText`)"
+        :maxlength="100"
+        @clear="resetSearch"
+        class="add-asset-token__search"
+      />
+      <div class="add-asset-token__switch-btn">
+        <s-switch v-model="showVerifiedAssetsOnly" :disabled="loading" />
+        <span>{{ t(`addAsset.${AddAssetTabs.Token}.switchBtn`) }}</span>
+      </div>
+      <asset-list :assets="foundAssets" class="asset-search-list" @click="handleSelectAsset">
+        <template #list-empty>
+          {{ t(`addAsset.${assetIsAlreadyAdded ? 'alreadyAttached' : 'empty'}`) }}
```

Comment:
Could this computed string be transfered to `AddAssetMixin`?

Reply:
They vary pretty much, don't think that is the case for mixins
<img width="582" alt="Screenshot 2022-05-05 at 12 27 44" src="https://user-images.githubusercontent.com/27702061/166879259-69aa5f10-fae2-48de-9a7d-05b90f38401a.png">
<img width="909" alt="Screenshot 2022-05-05 at 12 27 28" src="https://user-images.githubusercontent.com/27702061/166879264-ffced3a1-c31c-41cd-a092-d78ba3b70ab2.png">

Reply:
i was talked about message `{{ t(`addAsset.${assetIsAlreadyAdded ? 'alreadyAttached' : 'empty'}`) }}`
Btw the filter callback better transfer to mixin, to be sure that filtering logic is the same

Reply:
moved both

Reply:
I rewrote `assetIsAlreadyAdded` (which caused a bug), so, set of assets is different in each tab; no longer needs to pass to mixins now